### PR TITLE
Adds another 'Active' filter users and admins in AdminConsole/UserManagement/Users

### PR DESCRIPTION
**Pull request by tmotyl**
#### Summary
Mattermost system console user list should allow to filter out inactive users, or in other words to only show active users and admins.

Right now you only have following options:
"All users"
"System Admins"
"Guests"
"Inactive"

This PR adds new option "Active" to the list.
The API on the server side is already present:
This URL returns inactive  users:
/api/v4/users?page=0&per_page=100&inactive=true

This URL returns active users:
/api/v4/users?page=0&per_page=100&inactive=false

#### Ticket Link

https://mattermost.uservoice.com/forums/306457-general/suggestions/44133915-filter-out-inactive-users-in-mattermost-system-con

and partially https://github.com/mattermost/mattermost-server/issues/12319

#### Related Pull Requests

- Has server changes (please link here)
- Has mobile changes (please link here)

#### Release Note

```release-note
Adds another 'Active' filter users and admins in AdminConsole/UserManagement/Users
```

### CODE REVIEW

**Comment by M-ZubairAhmed** [comment]
thank you for the PR @tmotyl can we have a discussion in https://community.mattermost.com to coordinate developers and PMs opinions on this feature
cc @jwilander

**Comment by jwilander** [comment]
We chatted about this on community.mattermost.com. It's a minor improvement so falls under our policy where a ticket isn't required. Added @laneycs for PM review and @BenCookie95 for second dev review.

Thanks @tmotyl !

**Review by laneycs — APPROVED** [review]
[👍🏽](https://emojipedia.org/thumbs-up-medium-skin-tone/) Passes PM review.

File: components/admin_console/system_users/system_users.tsx
Comment (M-ZubairAhmed) [inline components/admin_console/system_users/system_users.tsx:357]: nit : Can we move active before inactive?

**Review by M-ZubairAhmed — CHANGES_REQUESTED** [review]

**Review by M-ZubairAhmed — APPROVED** [review]
Actually i am gonna approve since my previous comment is non blocking

**Comment by tmotyl** [comment]
here you go @M-ZubairAhmed

File: components/admin_console/system_users/system_users.tsx
Comment (tmotyl) [reply components/admin_console/system_users/system_users.tsx:356]: done

**Review by BenCookie95 — APPROVED** [review]
This is great, thanks @tmotyl!

**Comment by jgilliam17** [comment]
Hi @tmotyl can you please update the branch? Thanks 🙂

**Comment by tmotyl** [comment]
@jgilliam17 that's strange, I always make sure the checkbox about maintainers being able to edit my branch is checked when I'm contributing.
Anyway, I've updated the branch by merging upstream changes. Is it ok, or do you want me to rebase branch on top of master?

**Comment by jgilliam17** [comment]
Thanks @tmotyl I think this is sufficient. 
@M-ZubairAhmed do you mind checking the running workflow and approving if ok? thanks 🙂

**Comment by jgilliam17** [comment]
Thanks @tmotyl and apologies for delay in getting to this PR
- [E2E report](https://automation-dashboard.vercel.app/cycles/437cd1cc-1b41-413a-a639-650141cfc15a) looks good, no PR related failures
I am seeing one issue
Deactivated user consistently sorts as "active'
Screen Shot 2022-03-22 at 2 21 53 PM

**Comment by tmotyl** [comment]
changed api param from
/api/v4/users?page=0&per_page=100&inactive=false 
To 
/api/v4/users?page=0&per_page=100&active=true

Will @mm-cloud-bot pick up the new commit automatically, or does one need to trigger it?

**Comment by tmotyl** [comment]
@jgilliam17 Please try it now.

**Comment by jwilander** [comment]
Thanks for the contribution @tmotyl !

**Pull request merged**
